lib/runtime-core/src/backend.rs

```diff
@@ -104,22 +50,6 @@ pub struct InlineBreakpoint {
     pub ty: InlineBreakpointType,
 }
 
-#[cfg(test)]
-mod backend_test {
-    use super::*;
-    use std::str::FromStr;
-
-    #[test]
-    fn str_repr_matches() {
-        // if this test breaks, think hard about why it's breaking
-        // can we avoid having these be different?
-
-        for &backend in &[Backend::Cranelift, Backend::LLVM, Backend::Singlepass] {
-            assert_eq!(backend, Backend::from_str(backend.to_string()).unwrap());
-        }
-    }
-}
-
```

Comment:
Looks like these tests didn't survive the migration -- looks like we should keep it around as long as we have an enum which can get out of date

Reply:
This is now solved :)